src/base/bittorrent/session.cpp

```diff
@@ -1347,6 +1361,48 @@ bool Session::addTorrent_impl(AddTorrentData addData, const MagnetUri &magnetUri
     return true;
 }
 
+bool Session::findIncompleteFiles(TorrentInfo &torrentInfo, QString &savePath) const
+{
+    auto findInDir = [](const QString &dirPath, TorrentInfo &torrentInfo) -> bool
+    {
+        bool found = false;
+        if (torrentInfo.filesCount() == 1) {
+            const QString filePath = dirPath + torrentInfo.filePath(0);
+            if (QFile(filePath).exists()) {
+                found = true;
+            }
+            else if (QFile(filePath + QB_EXT).exists()) {
+                found = true;
+                torrentInfo.renameFile(0, torrentInfo.filePath(0) + QB_EXT);
```

Comment:
@glassez I am not sure on who you are responding but I was responding to @evsh and my comment was meant to say "libtorrent will take care of it".